ntrans.py

```diff
@@ -1,107 +1,75 @@
-import nltk
 import translatepy
-import re
-import collections
 import csv
-# import ssl
 
+"""
+DEVELOPER INFO:
 
-def prepare_corpus_data(data_size=1000000):
-    # TODO: Put code that downloads BNC here
-
-    bnc_corpus = nltk.corpus.BNCCorpusReader(root='BNC/Texts/', fileids=r'[A-K]/\w*/\w*\.xml')
-
-    bnc_sents = []
-
-    # TODO:
-    # If user wants a smaller data set than the full corpus, it would be better
-    # to pick random sents() from bnc_corpus instead of for looping over the first
-    # X sents. If data_size = 10.000, pick 10.000 random sents from bnc_corpus
-
-    for count, sentence in enumerate(bnc_corpus.sents()):
-        string_sent = ' '.join(sentence)
-        bnc_sents.append(string_sent)
-        if count == 1000000:  # TODO: Change count == 10000 to "count == data_size"
-            print(count)
-            break
-        # TODO: Remake print(count) to progress bar in tkinter
-        if count % 1000 == 0:
-            print(count)
+The usage of this script relies on the N-gram files in "./ngrams"
 
-    return ' '.join(bnc_sents)
+These files are created through ntrans_dataprep.py
+"""
 
 
 def create_csv_file(source_target_pairs, save_path=None):
+    """
+    Writes source/target pairs to a csv-file
+    """
```

Comment:
This is **correct** usage of `"""`: this is a docstring, and not a comment. Because it is in the beginning of the function, it actually shows up in `help()` output:

```
>>> from ntrans import create_csv_file
>>> help(create_csv_file)
>>> Help on function create_csv_file in module ntrans:

create_csv_file(source_target_pairs, save_path=None)
    Writes source/target pairs to a csv-file

```